docs/tutorials/menu.md

```diff
@@ -0,0 +1,15 @@
+<!--[metadata]>
++++
+aliases = [
+]
+title = "Learn by example"
+description = "Explains how to work with containers"
+keywords = ["docker, introduction, documentation, about, technology, docker.io, user, guide, user's, manual, platform, framework, home,  intro"]
```

Comment:
@SvenDowideit Okay, but I don't see `type - "menu"` in the Piñata `menu.md`s you created ...